Add Flex logs to best practices (#24352)
DataDog/documentation: content/en/logs/guide/best-practices-for-log-management.md

Setting daily quotas on your indexes can help prevent billing overages when new log sources are added or if a developer unintentionally changes the logging levels to debug mode. See [Alert on indexes reaching their daily quota](#alert-on-indexes-reaching-their-daily-quota) on how to set up a monitor to alert when a percentage of the daily quota is reached within the past 24 hours.

### Set up storage for long-term retention

> If you want to retain logs long-term with the ability to query them as needed, without the cost of standard indexes configure [Flex Logs][30]. This tier decouples storage from compute costs so you can cost-effectively retain logs that may not need to be queried as frequently as standard indexes.

Reviewer: ```suggestion
If you want to retain and query logs for an extended time, but without the cost of standard indexes, configure [Flex Logs][30]. This tier decouples storage from compute costs so you can cost-effectively retain logs that might not need to be queried as frequently as standard indexes.
```
